src/visualization/interaction/OrbitControls.js

```diff
@@ -14,7 +20,9 @@
  * @param userZoomSpeed (optional) - the speed for zooming
  * @param userRotateSpeed (optional) - the speed for rotating
  * @param autoRotate (optional) - if the orbit should auto rotate
- * @param autoRotate (optional) - the speed for auto rotating
```

Comment:
This param is actually typo for `autoRotateSpeed`. Could you rename `autoRotate` to `autoRotateSpeed` and put it back?